README.md

```diff
@@ -1,3 +1,8 @@
+> [!WARNING]
+> As of the 1.22.3 release, this repo is no longer being used to
+> release leveldb for CocoaPods. Instead, refer to
+> https://github.com/firebase/leveldb/tree/firebase-release.
+
 # LevelDB Library podspec
 
 This repository provides a podspec to create a CocoaPod for the
```

Comment:
Also should move the contents below line 11 to the other repo in a CocoaPods build section .

Reply:
Done in https://github.com/firebase/leveldb/pull/8